src/main/java/com/onarandombox/MultiverseCore/utils/settings/MVSettings.java

```diff
@@ -0,0 +1,232 @@
+package com.onarandombox.MultiverseCore.utils.settings;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.List;
+import java.util.logging.Logger;
+
+import com.onarandombox.MultiverseCore.utils.settings.migration.ConfigMigrator;
+import io.github.townyadvanced.commentedconfiguration.CommentedConfiguration;
+import io.github.townyadvanced.commentedconfiguration.setting.CommentedNode;
+import io.github.townyadvanced.commentedconfiguration.setting.TypedValueNode;
+import io.github.townyadvanced.commentedconfiguration.setting.ValueNode;
+import org.bukkit.plugin.Plugin;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+public class MVSettings {
```

Comment:
Not crazy about having a thing called Settings and a thing called Config. How are they different and how do we easily understand that? If `MVConfig` is the primary place to get settings, what if we call this one something like `ConfigHandle`?